notebooks/samples/explanations/tf2/ai-explanations-image.ipynb

```diff
@@ -95,7 +95,7 @@
       "source": [
```

Comment:
<p>Missing . at end of first paragraph: "...and fill in most of the fields."</p><p> </p><p>Second paragraph, first sentence: "First, you instantiate a metadata builder object..."</p><p> </p><p><code>input_layer</code>: Tells AI Explanations...</p><p><code>input_baselines</code> Sets a non-informative baseline to compare against. For image models... This example uses <code>np.random</code>...﻿</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>